sources/src/cache-base.ts

```diff
@@ -239,6 +243,23 @@ export class GradleStateCache {
         return fs.readFileSync(absolutePath, 'utf8')
     }
 
+    /**
+     * When the GitHub environment ACTIONS_RUNNER_DEBUG is true, run Gradle with --info and --stacktrace.
+     * see https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging
+     */
+    configureInfoLogLevel(): void {
+        const infoProperties = `org.gradle.logging.level=info\norg.gradle.logging.stacktrace=all\n`
+        const propertiesFile = path.resolve(this.gradleUserHome, 'gradle.properties')
+        if (fs.existsSync(propertiesFile)) {
+            core.info(`Merged --info and --stacktrace into existing ${propertiesFile} file`)
+            const existingProperties = fs.readFileSync(propertiesFile, 'utf-8')
```

Comment:
Starting with Java 9, utf-8 is used by default, otherwise latin-1. What do you prefer?

Reply:
I prefer 'utf-8'.